cli/pcluster/easyconfig.py

```diff
@@ -252,23 +279,23 @@ def configure(args):  # noqa: C901 FIXME!!!
     keys = list_keys(aws_region_name)
     key_name = prompt(
         "Key Name",
-        get_config_parameter(config, section=cluster_label, parameter_name="key_name", default_value=keys[0]),
+        get_config_parameter(config, section=cluster_label, parameter_name="key_name", default_value=keys[0][0]),
         options=keys,
-        check_validity=True,
     )
 
+    vpcs = list_vpcs(aws_region_name)
     vpc_id = prompt(
         "VPC ID",
-        get_config_parameter(config, section=vpc_label, parameter_name="vpc_id", default_value=None),
-        options=list_vpcs(aws_region_name),
-        check_validity=True,
+        get_config_parameter(config, section=vpc_label, parameter_name="vpc_id", default_value=None) or vpcs[0][0],
```

Comment:
Same as before --> vpcs[0][0] --> default_vpc_id

Then, could you pass default_vpcs_id as default_value to the get_config_parameter?